client/keys.go

```diff
@@ -388,7 +388,8 @@ func (k *Key) Unseal(in *pb.SealedBytes, opts UnsealOpts) ([]byte, error) {
 		// ECC signing key.
 		// We can detect this bug, as it triggers a RCInsufficient
 		// Unmarshaling error.
-		if paramErr, ok := certErr.(tpm2.ParameterError); ok && paramErr.Code == tpm2.RCInsufficient {
+		var handleError tpm2.HandleError
+		if errors.As(certErr, &handleError) && handleError.Code == tpm2.RCInsufficient {
```

Comment:
Can you also still include the old ParameterError check in the condition? Seem like different TPM implementations will return slightly different error code on this one.

Reply:
I've added ParameterError, but left `error.As` pattern, because in my opinion this check will be more future-proof, at somepoint someone could wrap error above.